style.css

```diff
@@ -1,54 +1,66 @@
 body {
   margin: 0;
 }
+
 .wrapper {
   position: relative;
   padding: 0 40px;
   margin-left: auto;
   margin-right: auto;
   max-width: 1530px;
 }
+
 .header {
   display: flex;
-  justify-content: space-between;  
+  justify-content: space-between;
+  margin-bottom: 100px;  
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)